src/styles/theming/sap_fiori_3_dark.scss

```diff
@@ -265,17 +266,11 @@
   /* Table */
   --fdTable_Hover_Down_Color: inherit;
 
-  /* Link */
-  --fdLink_Text_Decoration_Opposite: underline;
-  --fdLink_Text_Decoration: none;
-  --fdLink_Text_Focus_Color: var(--sapLinkColor);
-  --fdLink_Focus_Background: transparent;
-  --fdLink_Focus_Border_Radius: 0;
-  --fdLink_Focus_Text_Shadow: none;
-  --fdLink_Hover_Outline_Offset: -0.0625rem;
-  --fdLink_Hover_Outline_Width: var(--sapContent_FocusWidth);
-  --fdLink_Hover_Outline_Color: var(--sapContent_FocusColor);
-  --fdLink_Hover_Outline_Style: var(--sapContent_FocusStyle);
+  /* Message Box */
+  --fdMessage_Box_Error_Box_Shadow: inset 0 -0.0625rem var(--sapErrorBorderColor), var(--sapContent_HeaderShadow);
```

Comment:
Were these message box changes committed to this branch accidentally?

Reply:
Yes, good thing you noticed, removed